delegator/src/sign/helper/utils.ts

```diff
@@ -0,0 +1,65 @@
+import { Transaction } from '@prisma/client'
+import Caver from 'caver-js'
+import { DelegatorError, DelegatorErrorCode } from './errors'
+import { SignatureData } from 'caver-js'
+import { SignTxData } from './types'
+
+const PROVIDER_URL = 'https://api.baobab.klaytn.net:8651'
+const caver = new Caver(PROVIDER_URL)
```

Comment:
I think everything in this file should be moved to `sign.service.ts` file.

The following parts should be inside of constructor.

```typescript
const caver = new Caver(PROVIDER_URL)
const feePayerKeyring = caver.wallet.keyring.createFromPrivateKey(process.env.SIGNER_PRIVATE_KEY)
caver.wallet.add(feePayerKeyring)
```